### DISCUSSION TITLE
Can the Projects page be hidden in the Buildbot UI?

### DISCUSSION THREAD

**Discussion by muks** [buildbot/buildbot#7923] [General]
Hello. We don't use projects (and I suspect it's currently the case for a lot of Buildbot users). The ProjectsView is empty. Can the "Projects" menu item not be added to the sidebar menu if there are no projects configured?

Alternatively, is there any way to hide it via configuration?

**Comment by p12tic** [comment]
This is a known bug unfortunately.

**Comment by bartvdbraak** [comment]
Any updates on this?
